components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+                        <div className="px-4 self-center w-1/12">
+                            <div className={"rounded-full w-3 h-3 text-sm align-middle bg-green-500"}>
+                                &nbsp;
+                        </div>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-3/12">
+                            <span className="my-auto font-medium truncate overflow-ellipsis">{getPlan(sub)?.name}</span>
+                            <span className="text-sm my-auto text-gray-400 truncate overflow-ellipsis">Purchased on {formatDate(sub?.startDate)}</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-2/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{slots.filter(s => s.teamSubscription.id === sub.id).length || "–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Members</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-5/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{"–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Next Biling Cycle</span>
+                        </div>
+                        <div className="my-auto flex w-1/12 pl-8">
+                            <div className="self-center hover:bg-gray-200 rounded-md cursor-pointer w-8">
+                                <ContextMenu menuEntries={subscriptionMenu(sub)}>
+                                    <img className="w-8 h-8 p-1" src={ThreeDots} alt="Actions" />
+                                </ContextMenu>
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+        )}
+
+
+    </div>);
+}
+
+function InviteMembersModal(props: {
+    sub: TeamSubscription,
+    onClose: () => void
+}) {
+
+    const getInviteURL = () => {
+        const link = new URL(window.location.href);
+        link.pathname = '/subscription/'
+        link.search = '?teamid=' + props.sub.id;
+        return link.href;
+    }
+
+    const copyToClipboard = (text: string) => {
+        const el = document.createElement("textarea");
+        el.value = text;
+        document.body.appendChild(el);
+        el.select();
+        try {
+            document.execCommand("copy");
+        } finally {
+            document.body.removeChild(el);
+        }
+    };
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">Invite Members</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4 space-y-2">
+            <p className="pb-4 text-gray-500 text-base">Invite members to the team using the URL below.</p>
```

Comment:
**suggestion**: After adding the top padding on each group, we could remove some bottom padding here.

```suggestion
            <p className="pb-2 text-gray-500 text-base">Invite members to the team using the URL below.</p>
```